### DISCUSSION TITLE
Submission to JOSS - the Journal of Open Source Software

### DISCUSSION THREAD

**Discussion by StevenClontz** [pi-base/data#614] [Show and tell]
Feedback on my/@jamesdabbs [submission](https://github.com/pi-base/data/files/14889226/paper.pdf) to https://joss.theoj.org/ is appreciated. Especially from @jamesdabbs. (-: Source: https://github.com/pi-base/web/blob/joss2024/paper.md

[Updated PDF 2024-04-05](https://github.com/pi-base/data/files/14889226/paper.pdf)

**Comment by katjabercic** [comment]
I'm not familiar with the journal's format - would it make sense to make some of the requirements and design choices explicit? I.e., why did you go with a single-page, client-side application (as opposed to avoiding Javascript, and the "generally easier" server side)?

**Reply by StevenClontz** [reply]
Nice thought - I'll add that. (The answer: servers have to be maintained and can go down - Javascript apps run on the client and can be deployed to any static web host.)

**Reply by StevenClontz** [reply]
updated now

**Comment by prabau** [comment]
> "recent enhancements to the web inferface now make the discovery of open (to π-Base) questions,"

something missing at the end of that?

Also, typos in two of the last names in the Acknowledgments section.

**Reply by StevenClontz** [reply]
addressed now, thanks!

**Comment by StevenClontz** [comment]
Note: see if I can find analytics

**Reply by StevenClontz** [reply]
✔️

**Comment by StevenClontz** [comment]
add Buzzard citation

**Reply by StevenClontz** [reply]
✔️

**Comment by StevenClontz** [comment]
Last call for feedback - I hope to submit early next week.

**Reply by ccaruvana** [reply]
Sorry for not dropping in sooner. I think it looks great. 👍🏼

**Comment by jamesdabbs** [comment]
I wasn't familiar with JOSS, but from spot checking their About section and a couple articles, this LGTM. :shipit:

**Comment by StevenClontz** [comment]
This paper is now under review: https://github.com/openjournals/joss-reviews/issues/6594
